src/main/java/org/cactoos/list/ListIteratorNoNulls.java

```diff
@@ -95,11 +95,17 @@ public void remove() {
 
     @Override
     public void set(final T item) {
+        if (item == null) {
```

Comment:
@victornoel Why it didn't require this checks before?

Reply:
@andreoss I don't know but it was a mistake IMHO. I've removed my changes and added a todo instead so that some tests are also added.